fv3core/decorators.py

```diff
@@ -103,6 +104,7 @@ def __init__(
             domain: gt4py domain to use at call time
             stencil_config: container for stencil configuration
             externals: compile-time external variables required by stencil
+            skip_passes: compiler passes to skip when building stencil
```

Comment:
Can we get a comment here to note that this should be a temporary solution has we introduce a config system